pjsip/src/pjsua-lib/pjsua_vid.c

```diff
@@ -994,6 +994,8 @@ static pj_status_t setup_vid_capture(pjsua_call_media *call_med)
     /* Done */
     inc_vid_win(wid);
     call_med->strm.v.cap_win_id = wid;
+    PJ_LOG(4,(THIS_FILE, "Call %d: video capture set up with dev %d, wid=%d",
+	      call_med->call->index, call_med->strm.v.cap_dev, wid));
```

Comment:
Perhaps better with call index & media index info, usually in the beginning of line: "Call %d media %d: ... " (check other logs)? Also all new logs in the same file below.

Reply:
which new logs? This is the only one in this file.